doc/fuzzing.md

```diff
@@ -127,15 +127,23 @@ Patience is useful; even with improved throughput, libFuzzer may need days and
 
 ## Reproduce a fuzzer crash reported by the CI
 
-- `cd` into the `qa-assets` directory and update it with `git pull qa-assets`
 - locate the crash case described in the CI output, e.g. `Test unit written to
   ./crash-1bc91feec9fc00b107d97dc225a9f2cdaa078eb6`
+- create a file containing the seed reported right after the crash, e.g. `
+  Base64: Nb6Fc/97AACAAAD/ewAAgAAAAIAAAACAAAAAoA==`
 - make sure to compile with all sanitizers, if they are needed (fuzzing runs
   more slowly with sanitizers enabled, but a crash should be reproducible very
   quickly from a crash case)
-- run the fuzzer with the case number appended to the seed corpus path:
-  `FUZZ=process_message build_fuzz/bin/fuzz
-  qa-assets/fuzz_corpora/process_message/1bc91feec9fc00b107d97dc225a9f2cdaa078eb6`
+- run the fuzzer specifying the crash path:
+  `FUZZ=process_message build_fuzz/bin/fuzz ./crash-1bc91feec9fc00b107d97dc225a9f2cdaa078eb6`
```

Comment:
+1 I don't think it makes sense to remove CI docs.

Maybe what you're trying to write is something for developer-notes.md that is a hint for what to do when someone hands you a fuzz input that looks like "Nb6Fc/97AACAAAD/ewAAgAAAAIAAAACAAAAAoA==" ?

Reply:
Updated to add this, plus an example as the last bullet point